ext/ctype/ctype.stub.php

```diff
@@ -2,24 +2,51 @@
 
 /** @generate-class-entries */
 
+/**
+ * @compile-time-eval
+ */
 function ctype_alnum(mixed $text): bool {}
 
+/**
+ * @compile-time-eval
+ */
 function ctype_alpha(mixed $text): bool {}
 
+/**
+ * @compile-time-eval
+ */
 function ctype_cntrl(mixed $text): bool {}
 
+/**
+ * @compile-time-eval
+ */
 function ctype_digit(mixed $text): bool {}
 
 function ctype_lower(mixed $text): bool {}
```

Comment:
Why was this skipped?

Reply:
because ctyle lower/upper still relies on locale

Reply:
Ah, thanks!